src/util/address.rs

```diff
@@ -124,6 +127,7 @@ impl std::error::Error for Error {
             | InvalidSegwitV0ProgramLength(_)
             | UncompressedPubkey
             | ExcessiveScriptSize
+            | InvalidAddressType(_)
             | UnrecognizedScript => None,
```

Comment:
Can we have `InvalidAddressType` last so that its in the same order as in the enum please.

Reply:
Yes! Changed :+1: